e2e/entries.e2e-spec.ts

```diff
@@ -9,9 +9,9 @@ describe('Entries E2E Test', () => {
 
   let originalTimeout;
   let loginPage = new LoginPageObject();
```

Comment:
Thanks correct i "fixed" those in the review  and realized they have been initalizied twice